src/options/bv_bitblast_mode.h

```diff
@@ -88,13 +88,35 @@ enum BvProofFormat
   BITVECTOR_PROOF_LRAT,
 };
 
+/**
+ * When the BV solver does eager bitblasting backed by CryptoMiniSat, proofs
```

Comment:
```suggestion
 * When the BV solver does eager bit-blasting backed by DRAT-producing SAT solvers, proofs
```

We will also support CaDiCaL in the future.